.gitignore

```diff
@@ -1,5 +1,6 @@
 target/
 /node_modules
+/MyApp-*
```

Comment:
That looks like placeholder, probably need real name here

Reply:
Yes, but `MyApp-<os>-<arch>` is the name we will get after build at the moment. Will change all occurrences when we have name :)